mozregression/cli.py

```diff
@@ -265,6 +265,12 @@ def create_parser(defaults):
                              " release using a specific release number, and"
                              " a nightly using a specific buildid")
 
+    parser.add_argument('-P', '--process-output', choices=('0', '1'),
+                        default=defaults['process-output'],
+                        help=("Manage process output logging. 1 to enable, 0"
```

Comment:
I feel like `process output logging` is not entirely clear. I would at least say `Manage process output (stdout) logging.`

Also, why isn't this a store_true argument? I don't see why specifying 1 or 0 on the command line should really be necessary.